portfolio/portfolio.go

```diff
@@ -178,7 +179,7 @@ func (b *Base) UpdateAddressBalance(address string, amount float64) {
 func (b *Base) RemoveExchangeAddress(exchangeName string, coinType currency.Code) {
 	for x := range b.Addresses {
```

Comment:
🚧 Hidden DeleteFunc here.
```suggestion
	b.Addresses = slices.DeleteFunc(b.Addresses, func(a Address) bool {
		return a.Address == exchangeName && a.CoinType.Equal(coinType)
	})
```